core/arch/arm/plat-vexpress/conf.mk

```diff
@@ -54,6 +54,7 @@ endif
 
 CFG_WITH_STATS ?= y
 CFG_ENABLE_EMBEDDED_TESTS ?= y
+CFG_CORE_BGET_BESTFIT ?= y
```

Comment:
Enable BestFit or increase the heap size?

Reply:
I'll try to increase the heap also.